env.example

```diff
@@ -6,3 +6,6 @@ PORT=8080
 
 # REDIS_URL specifies Redis server info
 REDIS_URL=redis://127.0.0.1:6379
+
+#GITHUB_TOKEN is used when fetching data from the GitHub API
+GITHUB_TOKEN=123456789
```

Comment:
Leave the value blank, and add a link in your comment to docs on how to get one of these and what it is.  Also, add a space after the `#` in your comment above.